setup.py

```diff
@@ -132,7 +132,7 @@ def find_version(*file_paths):
 ]
 
 TORCH_EXTRAS = [
-    "torch>=1.9.1,<1.14;python_version < '3.11'",
+    "torch>=1.13.0,<2.1;python_version < '3.11'",
```

Comment:
If some NNCF user already has it installed with older PT version and then installs a newer NNCF version with your changes, will it fail or what will happen?